backend/test/controllers/game_configurations_controller_test.rb

```diff
@@ -35,4 +53,59 @@ class GameConfigurationsControllerTest < ActionDispatch::IntegrationTest
 
     assert_response 204
   end
+
+  test "should not be able to create invalid input_sets: too few strings" do
+    local_game_configuation = GameConfiguration.new(
+      :name => 'Test2',
+      :num_matches => 1,
+      :input_set => 'test2 test2'
+    )
+    post game_configurations_url, params: { game_configuration: { input_set: local_game_configuation.input_set, name: local_game_configuation.name, num_matches: local_game_configuation.num_matches } }, as: :json
+    
+    assert_response 422
```

Comment:
We probably shouldn't allow them to have multiple of the same string in the input either. 

For example, `:input_set => 'test test test'` should also fail, because which test is which?